doc/source/examples/dem/dem.rst

```diff
@@ -12,9 +12,11 @@ In `Example 4 <./rotating-drum-with-post-processing/rotating-drum-with-post-proc
 
 In `Example 5 <./rotation-of-box/rotation-of-box.html>`_ (rotation of box), we simulate the dynamics of particles in a rotating box. The key difference between `Example 3 <./rotating-drum/rotating-drum.html>`_ and `Example 5 <./rotation-of-box/rotation-of-box.html>`_ is that `Example 3 <./rotating-drum/rotating-drum.html>`_ uses a ``rotational`` boundary condition to simulate the rotating drum, while `Example 5 <./rotation-of-box/rotation-of-box.html>`_ uses a rotation of triangulation (``grid motion``). 
 
-In `Example 6 <./silo/silo.html>`_ (silo), we simulate the filling (using a ``floating wall``) and discharge of particles in a wedge-shaped silo. 
+In `Example 6 <./silo/silo.html>`_ (silo), we simulate the filling (using a ``floating wall``) and discharge of particles in a wedge-shaped silo.
 
-In `Example 7 <./3d-small-scale-rotating-drum/3d-small-scale-rotating-drum.html>`_ (small scale rotating drum (3D)), we simulate the behaviour of particles in a small scale rotating drum subjected to a rotational angular velocity over the cylinder. 
+In `Example 7 <./rectangular-hopper/rectangular-hopper.html>`_ (rectangular hopper), we simulate the discharge of particles in a rectangular hopper and we post-processed results with PyVista to get the mass discharge rate.
+
+In `Example 8 <./3d-small-scale-rotating-drum/3d-small-scale-rotating-drum.html>`_ (small scale rotating drum (3D)), we simulate the behaviour of particles in a small scale rotating drum subjected to a rotational angular velocity over the cylinder.
```

Comment:
But I often refer to the silo example since parameters and feature used are also the same, it would be weird to have it after the hopper in my opinion.

Reply:
I think 8, 6 and 7 should be good, what do you think?

Reply:
Good for me